packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -2416,10 +2439,11 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
             break;
           }
 
-          case RoapScenario.ANSWER:
+          case RoapScenario.ANSWER: {
```

Comment:
These 2 changes in lines 2442 and 2446 are not required right? None of the other cases have this?